qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -93,10 +104,15 @@ function expect_false()
 
 function get_pid()
 {
-    local ns=$1
+    local pool=$1
+    local ns=$2
+
+    if [ -z "${pool}" ]; then
+        pool=${POOL}
+    fi
```

Comment:
Maybe it's just me, but I dislike signatures with default arguments where you end up needing to pass them anyway in case there are additional arguments.  I'd drop this and change `get_pid` callers to always pass the appropriate pool.